group_vars/control_room_all.yml

```diff
@@ -0,0 +1,16 @@
+# === NFS Client
+
+nfs_client_packages:
+  - glusterfs-client
+  - glusterfs-common
+  - glusterfs-server
+
+nfsclient_autofs_glusterfs_server: localhost
+nfsclient_autofs_glusterfs_backup_server: lnls449-linux
+nfsclient_autofs_glusterfs_log_file: /var/log/gluster.log
+
+# autofs: auto.sirius
+nfsclient_autofs_extra:
```

Comment:
Mounting the glusterfs volume using autofs as it is more reliable than /etc/fstab
Using `nfsclient_autofs_extra` variable to be a little more succinct